jsapp/xlform/src/view.rowDetail.coffee

```diff
@@ -26,15 +26,23 @@ module.exports = do ->
       _.extend(@, viewRowDetail.DetailViewMixins[@model.key] || viewRowDetail.DetailViewMixins.default)
       @$el.addClass(@extraClass)
 
+      # identify details that are possibly translatable (useful for locking)
+      if @model.key.includes("::") or @model.key in ['label', 'hint', 'guidance_hint']
+        @$el.addClass('js-translatable-text-input')
```

Comment:
class is not referenced anywhere else, maybe leftover from previous changes?

Reply:
Ah right, another leftover. The manage translations restriction changed so many times :P